CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 This file is used to list changes made in each version of the yum-centos cookbook.
 
+## Unreleased
+
+### Added
+- Add various SIG repos
+- Suite and InSpec tests for vault recipe
+
+### Changed
+- Changed to using $releasever instead of using ``node['platform_version'].to_i``
```

Comment:
You might want to look back at the history of the various yum repos because we switched to this after a lot of bugs. I can't remember what the issue was off the top of my head, but there was a reason for using this.

Reply:
I believe you're referring to #26? I fail to see why you wouldn't run a system without centos-release installed. Can you? There really wasn't much feedback on why that was needed.